SofaKernel/modules/SofaCore/src/sofa/core/topology/Topology.h

```diff
@@ -243,6 +243,11 @@ struct TopologyElementInfo<Topology::Hexahedron>
 };
 
 
+extern const unsigned int edgesInTetrahedronArray[6][2];
+
+extern const unsigned int edgesInHexahedronArray[12][2];
+extern const unsigned int quadsInHexahedronArray[6][4];
+extern const unsigned int verticesInHexahedronArray[2][2][2];
```

Comment:
why not putting directly the declaration inside the .h?

Reply:
I believe that if you declare it directly in the header file, the linkage would be internal instead of external.